readthedocs.yml

```diff
@@ -1,11 +1,19 @@
-name: jupyterhub
-type: sphinx
```

Comment:
Did you remove these on purpose or accidentally/

Reply:
This is on purpose, as in RTD config version 2, the name setting is [per project and for per version anymore](https://docs.readthedocs.io/en/stable/config-file/v2.html#migrating-from-the-web-interface) and [sphinx got its own setting.](https://docs.readthedocs.io/en/stable/config-file/v2.html#sphinx)